internal/services/gateway/action/user.go

```diff
@@ -57,6 +57,18 @@ func (h *ActionHandler) GetUser(ctx context.Context, userRef string) (*cstypes.U
 	return user, nil
 }
 
+func (h *ActionHandler) GetUserOrgs(ctx context.Context, userRef string) ([]*csapitypes.UserOrgsResponse, error) {
+	if !h.IsUserLogged(ctx) {
```

Comment:
BTW we should change the various actions to move all authn/authz logic here and remove as much as possible from the api methods. Created issue https://github.com/agola-io/agola/issues/307

Reply:
ok I agree with you, but i thinks it's better to leave this for the issue 307. What do you think ? If you agree I'm ready to push this PR. thanks